cadc-postgresql-dev/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM fedora:40
+FROM fedora:42
 
-# external URLs; allow --build-arg switch
-ARG PGDG=https://download.postgresql.org/pub/repos/yum/reporpms/F-40-x86_64/pgdg-fedora-repo-latest.noarch.rpm
+# MAINTAINER deprecated but upstream fedora image still has it
+LABEL org.opencontainers.image.authors="pdowler.cadc@gmail.com" maintainer="pdowler.cadc@gmail.com"
 
-ARG PGSERVER="postgresql15-server-15.8-1PGDG.f40.x86_64 postgresql15-contrib-15.8-1PGDG.f40.x86_64"
-ARG PGSPHERE=https://ws-cadc.canfar.net/vault/files/pdowler/rpms/pgsphere15-1.4.2-1.fc40.x86_64.rpm
+# external URLs; allow --build-arg switch
+ARG PGDG=https://download.postgresql.org/pub/repos/yum/reporpms/F-42-x86_64/pgdg-fedora-repo-latest.noarch.rpm
+ARG PGSERVER="postgresql15-server postgresql15-contrib"
+ARG PGSPHERE=https://ws-cadc.canfar.net/vault/files/pdowler/rpms/pgsphere15-1.4.2-1.fc42.x86_64.rpm
```

Comment:
I thought we have a static directory at the CADC for rpms and other libraries. Not anymore?

Reply:
I want external people and the github CI to be able to build this image, so I am the provider of the pgsphere package (via vault).